app/blog/2017-09-28-polymer-3-spanish.md

```diff
@@ -0,0 +1,93 @@
+ 
+Migrar a npm hará que los paquetes de Polymer estén disponibles para los millones de usuarios con los que cuenta npm y permitirá que estos puedan utilizar más fácilmente otros paquetes del masivo ecosistema de npm.
+ 
+Esta ha sido una característica muy solicitada, pero hemos estado esperando hasta que encontramos una buena solución para soportar instalaciones planas de módulos y poder mantener sincronizados los paquetes tanto de Bower como de npm.
+ 
+El cliente npm de yarn proporciona soporte para instalaciones planas, lo que soluciona nuestro principal problema para utilizar npm.
+ 
+Tras considerar muchas opciones para la sincronización de los paquetes de Bower y npm, nos dimos cuenta de que mantener versiones paralelas es poco práctico y concluimos que lo ideal es dejar de utilizar bower en la 3.x y usar npm exclusivamente.
+
+
+## ¿Cuándo sucederá esto?
+
+Ya puedes utilizar la versión nueva del Polymer y los elementos, pero aún necesitamos juntar algunas piezas más para que la versión esté lista para producción. Por ahora disponemos de: 
+
+*   Versiones funcionales de la librería Polymer y sus elementos publicados en npm.
+*   La herramienta Polymer Modulizer para automatizar la conversión de tus proyectos que utilizan Bower y HTML Imports a npm y módulos de ES6.
+*   Apoyo para paquetes de npm en el comando `polymer serve` del CLI
+
+
+Sin embargo, necesitamos algunas cosas más antes de lanzar Polymer 3.0, tales como:
+
+*   Soporte de módulos ES6 en al menos un navegador,, incluyendo imports dinámicos (Safari 10.1 y Chrome 61 ya soportan módulos, pero no los imports dinámicos).
+*   Efectuar una evaluación más exhaustiva de performance – el soporte nativo a los módulos de ES6 aún está en siendo adoptado en los navegadores, y se ha evaluado  relativamente poco.
+*   Completar la actualización del Polymer CLI y las herramientas relacionadas para que funcionen con módulos de ES6 y npm.
+*   Escribir documentación, ejemplos y plantillas.
+*   Y por último (pero no por ello menos importante) , necesitamos tu feedback, comentarios y participación para ayudarnos a que Polymer Modulizer, Polymer 3.0 y los elementos compatibles con Polymer 3.0 estén listos para producción. También necesitaremos tu ayuda para conseguir que los elementos actuales se actualicen a la última versión de Polymer.
+
+
+## ¿Qué viene a continuación?
+
+Esperamos que la versión preview de Polymer 3.0 los emocione tanto como a nosotros, y queremos escuchar tus comentarios sobre ella a través del [Slack de Polymer](https://polymer-slack.herokuapp.com/) o en las redes sociales. [Hands on with the Polymer 3.0 preview](https://www.polymer-project.org/blog/2017-08-23-hands-on-30-preview.html) 
```

Comment:
Only one comment here: It's kind of funky to have this link appended to the end of the line with no punctuation.

It'd be good to add a little tag line, like: "For more details see Hands on with the Polymer 3.0 preview."